apps/editors/tests/test_views.py

```diff
@@ -2330,6 +2330,20 @@ def test_viewing_queue(self):
         data = json.loads(r.content)
         eq_(data[str(self.addon.id)], self.editor.display_name)
 
+    def test_platform_text(self):
```

Comment:
Pick a more meaningful name, something like `test_display_same_files_only_once` for example, and maybe add a docstring explaining that you're testing the display for multiples files that are the same.

Another way to test that would be to add a test to `https://github.com/mozilla/olympia/blob/master/apps/editors/tests/test_helpers.py`, and only test the helper itself (instead of testing a GET on the url. I guess this is fine though, it's a very broad functional test ;)